README.md

```diff
@@ -38,7 +38,12 @@ To unmount the editor instance, call its `destroy()` method:
 editor.destroy();
 ```
 
-For a complete overview of the API, please refer to the [lib/index.d.ts](https://github.com/ls1intum/Apollon/blob/master/lib/index.d.ts) file.
+For a complete overview of the API, please refer to the [lib/es6/index.d.ts](https://github.com/ls1intum/Apollon/blob/master/lib/index.d.ts) file.
```

Comment:
please double check the link here, we do not use master as branch any more

Reply:
I think this information is outdated. My best guess is that they want to link to the lib/apolllon-editor.d.ts file. But since the lib directory is inside the .gitignore (rightfully so), we can't really link to the file on github as far as I can see. I would just remove the github link here.

Reply:
ok, please remove the link then

Reply:
removed it